content/cnc-router.md

```diff
@@ -0,0 +1,50 @@
+# The CNC Router - [X-Carve](https://shop.inventables.com/products/x-carve-1)
+
+The X-Carve CNC router works on an XY plane with limited Z-depth, capable of handling dimensions of 1000mm x 1000mm x 50mm. It is specifically designed to cut wood, and in part due to our dust collection system, should not be used for other materials.  
+Contrary to a typical CNC machine, our CNC router is designed to move relatively slowly. It relies on tool paths for rounded edges or complex shapes.
+
+## ⚠️ Safety
+
+1. Always keep the router's door closed when power is supplied to it.
+2. Secure your workpiece; if it isn't secure, it could go flying. Use a fixture or at least two clamps on each side of your piece.
+3. The router collet holding the bit must be properly tightened.
+4. **Never** leave the CNC router unattended while it's running. Do not walk away, even for a bathroom break.
+5. Running a CAM program on the router is a complex process involving fixturing, homing, and execution. If you haven’t done this before, have somebody walk you through your first run.
+
```

Comment:
"Always run a test runs with the spindle off above the part to make sure you path looks correct and you have the correct offsets."

Reply:
Covered in the docs from my NYIC PR